package.json

```
@@ -29,7 +29,8 @@
     "@opentelemetry/instrumentation-redis": ">= 0.35.4 < 0.36",
     "@opentelemetry/instrumentation-redis-4": ">= 0.35.5 < 0.36",
     "@opentelemetry/instrumentation-restify": ">= 0.34.2 < 0.35",
-    "@opentelemetry/sdk-node": ">= 0.45.0 < 0.46",
+    "@opentelemetry/sdk-node": "^0.47",
+    "@opentelemetry/exporter-metrics-otlp-proto": "^0.47.0",
     "@prisma/instrumentation": ">= 5.4.0 < 5.6.0",
```

And we're back in OpenTelemetry dependency hell! It feels like our second home at this point.

```
> npm list @opentelemetry/api
@appsignal/nodejs@3.0.30 /Users/nlapresta/Code/appsignal-nodejs
├── @opentelemetry/api@1.7.0
...
└─┬ @prisma/instrumentation@5.5.2
  ├── @opentelemetry/api@1.6.0
```

Reminder that, due to some mind-blowing OpenTelemetry API overengineering, these two versions of `@opentelemetry/api` won't be able to share global state with each other -- or one of them will read the other one's global state, but the other one will refuse to read it, or something like that. I don't remember the details. Anyway, it results in trace context being lost for instrumentations using one of the two versions. Fun!

Updating the version bounds on `@prisma/instrumentation` to [a version that includes this commit](https://github.com/prisma/prisma/commit/249341f6cec6e0edec187d1b28cef0999e223f30) should fix the issue:

```suggestion
"@prisma/instrumentation": ">= 5.6.0 < 5.10.0",
```

---

By the way @luismiramirez, this is why the other bounds were specified so strictly -- to make sure we didn't accidentally install a version with a different `@opentelemetry/api` version. 

Could you change the bounds on `@opentelemetry/api`, `@opentelemetry/instrumentation-http` and `@opentelemetry/sdk-node` back to a strict version range on the current minor version? I know it sucks, but it's the solution we have to this problem at the moment. Otherwise, the moment that `@opentelemetry/api@1.8.0` is released, all hell breaks loose.

---

I think I got them all right. The rebase was a bit of a mess. Could you re-check, please?

---

The `@prisma/instrumentation` bound didn't change -- maybe that got lost on the rebase?

After changing the bound, the `package-lock.json` needs to be updated using `npm update`.